app/controllers/api/v1/backgrounds_controller.rb

```diff
@@ -1,8 +1,16 @@
 class Api::V1::BackgroundsController < ApplicationController
   def index
-    image_data = UnsplashService.new.get_background_url(params[:location])
-    photo = Photo.new(image_data)
+    if image_data[:results].empty?
+      render json: {status: 404, result: 'There are zero results for this location'}, status: 404
+    else
+      photo = Photo.new(image_data)
+      render json: BackgroundsSerializer.new(photo)
+    end
+  end
+
+  private
 
-    render json: BackgroundsSerializer.new(photo)
+  def image_data
```

Comment:
Moved into a private method since gathering the image's data information shouldn't be accessible to just anyone